meta-refkit/recipes-image/images/refkit-installer-image.bb

```diff
@@ -23,10 +23,14 @@ REFKIT_INSTALLER_UEFI_COMBO () {
         rootfs=$4
         output_mounted=
         output_mountpoint=
+        output_luks=
+        LUKS_NAME=rootfs
+        LUKS_PASSWORD=refkit
```

Comment:
LUKS_PASSWORD is basically an internal constant. Its value has to match the string in initramfs-framework-refkit-luks.

FIXED_PASSWORD=yes is what enables the use of that fixed password.

We could make it so that there is no default password in initramfs-framework-refkit-luks unless the developer configures one in his build, and the image-installer script could be changed so that this fixed password has to passed via the FIXED_PASSWORD env variable. Would that be better? It's harder to use, but perhaps that is how we want it to be ;-}